index.html

```diff
@@ -213,7 +213,14 @@ <h3>legend</h3>
                     out += "<a href='https://github.com/" + REPO_OWNER + "/" + REPO_NAME + "/issues/" + row.mozPositionIssue + "' title='position discussion'><span class='glyphicon glyphicon-book' aria-hidden='true'></span><span class='sr-only'>position discussion</span></a> "
                   }
                   if (row.mozBugUrl && row.mozBugUrl !== "") {
-                    out += "<a href='" + row.mozBugUrl + "' title='mozilla bug'><span class='glyphicon glyphicon-link' aria-hidden='true'></span><span class='sr-only'>mozilla bug</span></a> "
+                    // Allow either a string or an array of strings.
+                    var bugArray = row.mozBugUrl;
+                    if (!Array.isArray(bugArray)) {
+                      bugArray = [bugArray];
+                    }
+                    for (var bug of bugArray) {
+                      out += "<a href='" + bug + "' title='mozilla bug'><span class='glyphicon glyphicon-link' aria-hidden='true'></span><span class='sr-only'>mozilla bug</span></a> "
```

Comment:
```suggestion
                      out += `<a href='${bug}' title='mozilla bug'><span class='glyphicon glyphicon-link' aria-hidden='true'></span><span class='sr-only'>mozilla bug</span></a> `;
```
??  (We're really inconsistent with semi-colon usage here BTW.)

Reply:
I think the inconsistency is because Mark likes omitting them, and I always use them.  (In this case I didn't notice because I was just reindenting.)

Reply:
I use [standard](https://standardjs.com).

Reply:
Another great example of why calling something a standard doesn't mean that it is :)